### ISSUE TITLE
Image alt text not sent?

### ISSUE THREAD

**Issue by janboddez** [janboddez/share-on-mastodon#7]
When a Featured Image is sent along (shouldn't this be made optional?), its alt text seems missing.

**Comment by janboddez** [comment]
In an older version (pre inclusion in the WP,org repo), the image title was sent along. This got removed during the move to `wp_remove_post()` (from cURL). It's now been re-added (or rather, this time the image's alt text is passed on, instead of its title).
